spec/models/sample_spec.rb

```diff
@@ -59,4 +65,24 @@
       expect(sample.requests.length).to eq 2
     end
   end
+
+  context 'after create' do
```

Comment:
Does this test anything differently to the added test above? Not sure whether its needed.

Reply:
I wanted to have an assertion to test the cases before Rails type cast the enums. I will remove the first example (which is redundant).